hack/toolchain.sh

```diff
@@ -11,6 +11,10 @@ tools() {
     cd tools
     go mod tidy
     GO111MODULE=on cat tools.go | grep _ | awk -F'"' '{print $2}' | xargs -tI % go install %
+
+    if ! echo "$PATH" | grep -q "${GOPATH:-undefined}/bin\|$HOME/go/bin"; then
```

Comment:
My path is actually at $HOME/workspaces/go. Thoughts on removing this check entirely and just printing a friendly message that isn't exact, but can at least provide users a hint.

Reply:
I'm guessing `$HOME/workspaces/go` is what your `$GOPATH` is set to?  But if that weren't set for you, then `go` would default to `$HOME/go`